simulation_parameters/Constants.cs

```diff
@@ -449,6 +449,8 @@ public static class Constants
 
     public const int SUBMENU_CANCEL_PRIORITY = -1;
 
+    public const int DIALOG_WINDOW_CANCEL_PRIORITY = 1;
```

Comment:
Can you add a summary here? State that this is the priority used for the input system.